include/circt/Dialect/Arc/ArcOps.td

```diff
@@ -23,16 +23,12 @@ include "mlir/Interfaces/ControlFlowInterfaces.td"
 include "mlir/Interfaces/InferTypeOpInterface.td"
 include "mlir/Interfaces/SideEffectInterfaces.td"
 
-class ArcOp<string mnemonic, list<Trait> traits = []> :
-  Op<ArcDialect, mnemonic, traits>;
+class ArcOp<string mnemonic, list<Trait> traits = []>
+    : Op<ArcDialect, mnemonic, traits>;
```

Comment:
It seems some kind of auto-format kicked in and reformatted a lot of these files. It's a pain, but would it be possible to back these out? We try to follow a policy of not including format only changes alongside the meaningful changes.

Reply:
I'd love to not include them, is there a way to still pass CI ?

Reply:
Did clang-format try to format the .td files? I think we usually hand format these... I don't think our CI formatting checks look at .td files.

Reply:
It looks like your "Ran clang-format" commit changed a bunch of .td files. I don't think that is expected. It might be that the version of clang-format you have thinks it can format .td files? But if you back that change out, and run clang-format only on the .cpp and .h files, you should be lint-clean in CI without changing all the .td files (which should be ok in CI).

Reply:
Ah ok, will do.